ubuntu-advantage

```diff
@@ -94,102 +94,76 @@ Commands:
  enable-fips <token>       enable the FIPS PPA repository and install,
                            configure and enable FIPS certified modules
  disabe-fips               currently not supported
+
 EOF
+    exit 1
 }
 
+main() {
+    local command="$1"
```

Comment:
Since this is a function now (main), we should also declare the "remove_snap" local variable used later on.

Reply:
good catch, done